### ISSUE TITLE
AR Mode - No Camera Feed 

### ISSUE THREAD

**Issue by ghus86** [google/model-viewer#1137]
### Description

Trying to view a 3d model in augmented reality using model viewer
I am able to see the model but the camera feed doenst open it stays black

#### Live Demo
http://testzone.pt/

### Browser Affected

- [X] Chrome
- [ ] Edge
- [x] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [X] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions

- model-viewer: latest version using cdn
- three.js: rXX

**Comment by ghus86** [comment]
@cdata any tip about what can be causing this ?
I am using a Samsung S8 with Android 9

**Comment by cdata** [comment]
@ghus86 sorry for the delayed response; I'm happy to help if I can.

Can you confirm if you are using WebXR for AR, or Scene Viewer? Scene Viewer is a native app, so you will see a different panel in your task switcher when it launches.

**Comment by ghus86** [comment]
@cdata no problem about delay

I am just using the tag model viewer and adding this 

```
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.js"></script>
<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
```

**Comment by cdata** [comment]
@ghus86 there may be other problems, but the first one I noticed is that your site is not being served over SSL, and going to the `https://` version results in an error suggesting that you do not have a valid certificate. In order for Scene Viewer to work, you need to serve your content over SSL (`https://` with a valid certificate).

**Comment by ghus86** [comment]
@cdata ok thanks 
didnt know that was required

**Comment by cdata** [comment]
After syncing up with the Scene Viewer team, I'm told that the `https://` requirement will go away when we fix #1097

**Comment by ghus86** [comment]
@cdata Thanks for your time 
configuring the ssl fixed my problem.
